README.md

```diff
@@ -69,21 +69,31 @@ _Pre-requisite: OpenShift 4.10 or above_
 
```

Comment:
@jotak @stleerh 


I suggest to add explanation on all the flags in::

```
$ go run ./main.go --help
Usage of /tmp/go-build1418114459/b001/exe/main:
  -console-plugin-image string
        The image of the Console Plugin
  -ebpf-agent-image string
        The image of the eBPF agent
  -flowlogs-pipeline-image string
        The image of Flowlogs Pipeline
  -health-probe-bind-address string
        The address the probe endpoint binds to. (default ":8081")
  -kubeconfig string
        Paths to a kubeconfig. Only required if out-of-cluster.
  -leader-elect
        Enable leader election for controller manager. Enabling this will ensure there is only one active controller manager.
  -metrics-bind-address string
        The address the metric endpoint binds to. (default ":8080")
  -v    print version
  -zap-devel
        Development Mode defaults(encoder=consoleEncoder,logLevel=Debug,stackTraceLevel=Warn). Production Mode defaults(encoder=jsonEncoder,logLevel=Info,stackTraceLevel=Error) (default true)
  -zap-encoder value
        Zap log encoding (one of 'json' or 'console')
  -zap-log-level value
        Zap Level to configure the verbosity of logging. Can be one of 'debug', 'info', 'error', or any integer value > 0 which corresponds to custom debug levels of increasing verbosity
  -zap-stacktrace-level value
        Zap Level at and above which stacktraces are captured (one of 'info', 'error', 'panic').
  -zap-time-encoding value
        Zap time encoding (one of 'epoch', 'millis', 'nano', 'iso8601', 'rfc3339' or 'rfc3339nano'). Defaults to 'epoch'.

```

Reply:
I think it's not really for this end user doc (README.md) which is focused on running the operator, but maybe more for `DEVELOPMENT.md`.

Reply:
ok ... agree